policyengine_canada/variables/gov/provinces/ns/tax/income/income_assistance/eligibility/assets/ns_income_assistance_applicable_assets.py

```diff
@@ -0,0 +1,34 @@
+from policyengine_canada.model_api import *
+
+
+class ns_income_assistance_applicable_assets(Variable):
+    value_type = float
+    entity = Household
+    label = "Nova Scotia income assistance applicable assets"
+    unit = CAD
+    definition_period = YEAR
+    reference = (
+        "https://novascotia.ca/just/regulations/regs/esiaregs.htm#TOC2_4"
+    )
+    defined_for = ProvinceCode.NS
+
+    adds = "gov.provinces.ns.tax.income.income_assistance.eligibility.assets.applicable_assets"
+
+    # None of the following is an applicable asset:
```

Comment:
could you file an issue to exclude these asset types? doesn't have to be in this pr